includes/admin/class-wc-rest-upe-flag-toggle-controller.php

```diff
@@ -55,8 +53,7 @@ public function register_routes() {
 				'args'                => [
 					'is_upe_enabled' => [
 						'description'       => __( 'Determines if the UPE feature flag is enabled.', 'woocommerce-gateway-stripe' ),
-						'type'              => 'string',
```

Comment:
Converted this to use boolean values, so that we can leverage `WC_Stripe_Feature_Flags::is_upe_checkout_enabled()` directly